pkg/crd/markers/validation.go

```diff
@@ -284,3 +294,55 @@ func (m Nullable) ApplyToSchema(schema *v1beta1.JSONSchemaProps) error {
 	schema.Nullable = true
 	return nil
 }
+
+// Defaults are only valid CRDs created with the v1 API
+func (m Default) ApplyToSchema(schema *v1beta1.JSONSchemaProps) error {
+	// Convert versioned schema to unversioned form
+	marshalledSchema, err := json.Marshal(schema)
+	if err != nil {
+		return err
+	}
+	unversionedSchema := &apiext.JSONSchemaProps{}
```

Comment:
the internal types are not meant for public consumption and might become inaccessible to this code base any time in the future. Why do we need that?

Reply:
No good reason, just conforming to what was required by the methods I was trying to call.